resources/views/auth/passwords/email.blade.php

```diff
@@ -8,8 +8,17 @@
 
 <div align="container">
   <div align="center" class="p-5">
-    <img class="mb-4" src={{asset(env('LOGIN_LOGO_PATH', '/img/processmaker_login.png'))}}>
-     <h3>{{__('Forgot Your Password?')}}</h3>
+    @php
+      $loginLogo = \ProcessMaker\Models\Setting::getLogin();
+      $isDefault = \ProcessMaker\Models\Setting::loginIsDefault();
+      if ($isDefault) {
+          $class = 'login-logo-default';
+      } else {
+          $class = 'login-logo-custom';
+      }
+    @endphp
+    <img src={{$loginLogo}} alt="{{ config('logo-alt-text', 'ProcessMaker') }}" class="{{ $class }}">
```

Comment:
Is there a way we can make this a component? 
A lot of repetitive code. Keep it DRY.
The same comment goes for the other files.
The only line that wouldn't belong to the component is the line 21 from this file